src/index/hnsw/faiss_hnsw.cc

```diff
@@ -227,8 +236,24 @@ class BaseFaissRegularIndexNode : public BaseFaissIndexNode {
         return index->ntotal;
     }
 
+    int64_t
+    Size() const override {
+        if (index == nullptr) {
+            return 0;
+        }
+
+        // a temporary yet expensive workaround
+        faiss::cppcontrib::knowhere::CountSizeIOWriter writer;
+        faiss::write_index(index.get(), &writer);
```

Comment:
Aha, the assumption here is that the size of the serialized data is the same as the deserialized one. Is that a truth? Or mostly a truth

Reply:
I'm not sure I follow the question.

Reply:
NM